src/palm-react-native-ui-kit/screens/auth/RecoverAccountScreen.tsx

```diff
@@ -99,6 +125,71 @@ const RecoverAccountScreen = (): ReactElement => {
     }
   }
 
+  // const onPressConfirm = async (): Promise<void> => {
+  //   if (alphaConfig.config?.waitlist) {
+  //     const address = usePkey
+  //       ? new ethers.Wallet(privateKey).publicKey
+  //       : ethers.Wallet.fromMnemonic(seedPhrase.join(' ')).publicKey
+
+  //     const ret = alphaConfig.waitList?.includes(address.toLowerCase())
+  //     if (!ret) {
+  //       setShowWaitlistModal(true)
+  //       return
+  //     }
+  //   }
+
+  //   if (recoverType === 'resetPin') {
+  //     const k = usePkey
+  //       ? await PkeyManager.getPkey()
+  //       : (await PkeyManager.getMnemonic()).split(' ')
+
+  //     const match = usePkey
+  //       ? k === privateKey
+  //       : JSON.stringify(k) === JSON.stringify(seedPhrase)
+
+  //     if (match) {
+  //       toast.show(t('Auth.RecoverSeedVerifyPassToast'), {
+  //         color: 'blue',
+  //         icon: 'check',
+  //       })
+  //       navigation.replace(Routes.Pin, {
+  //         type: 'reset',
+  //         result: (result: boolean): Promise<void> => {
+  //           result && navigation.pop()
+  //           return Promise.resolve()
+  //         },
+  //         cancel: (): void => {
+  //           navigation.pop()
+  //         },
+  //       })
+  //     } else {
+  //       toast.show(t('Auth.RecoverSeedVerifyFailToast'), {
+  //         color: 'red',
+  //         icon: 'info',
+  //       })
+  //     }
+  //   } else {
+  //     navigation.navigate(Routes.Pin, {
+  //       type: 'set',
+  //       result: async (result: boolean): Promise<void> => {
+  //         if (result === true) {
+  //           setLoading(true)
+  //           setTimeout(async () => {
+  //             await onClickConfirm()
+  //             setLoading(false)
+  //             navigation.replace(Routes.Sign4Auth)
+  //           }, 100)
+  //         }
+
+  //         return Promise.resolve()
+  //       },
+  //       cancel: () => {
+  //         navigation.pop()
+  //       },
+  //     })
+  //   }
+  // }
```

Comment:
?

Reply:
I did this to revert the code after the alphatest was over.
But it won't matter delete it.